### PR TITLE
[WIP] Add docker files to run the project simply local

### PR DESCRIPTION
**Pull request by mat1th**
I've add some docker files to easily run the project on any machine that has docker on it.

### CODE REVIEW

**Comment by Shifterovich** [comment]
Why would you run an HTML website in Docker? If you want live preview, just use `python -m SimpleHTTPServer` and go to 127.0.0.1:8000.

File: README.md [hunk 1]
```diff
@@ -33,22 +33,38 @@ We're trying to keep it simple and promote the best tools, not all of them. Maxi
...
+## Images
 
 - Provider logos should be exactly 200 x 70 ([example](https://www.privacytools.io/img/provider/AirVPN.gif))
 - Tool logos should be exactly 120 x 120 ([example](https://www.privacytools.io/img/tools/ChatSecure.png))
 
-#Support privacytools.io
+# Run local using Docker
+You can simple run this project local or on your server with Docker. Run ```docker-compose up```. Yours Apache server is now running.
+
+The site is acceible local on [localhost:8180](http://localhost:8180)
```
Comment (kav2k) [inline README.md]: Typo

File: README.md [hunk 2]
```diff
@@ -33,22 +33,38 @@ We're trying to keep it simple and promote the best tools, not all of them. Maxi
...
+
+## Variables
+There are some variables you can change in the ```docker-compose.yml```. Over here are the variables.
```
Comment (Shifterovich) [inline README.md]: Not a very good formulation.

**Comment by mat1th** [comment]
@Shifterovich Yes you can also use python. But I thought it would be easy to launch the project with docker. When you think it doen't add any value I will close the pull request.

**Comment by Shifterovich** [comment]
I'll let @privacytoolsIO decide.

**Pull request closed without merging**